bionic/decorators.py

```diff
@@ -86,6 +86,28 @@ def persist(enabled):
     return provider_wrapper(AttrUpdateProvider, 'should_persist', enabled)
 
 
+def memoize(enabled):
+    """
+    Indicates whether computed values should be cached memoized
```

Comment:
nit: we should probably say something more explicit, like "should be cached in memory."

Reply:
Agreed